content/en/docs/releases/supported-releases/index.md

```diff
@@ -51,15 +51,21 @@ current `<minor>` release. A patch is usually a small change relative to the `<m
 
 ## Support status of Istio releases
 
-| Version         | Currently Supported   | Release Date        | End of Life          | Supported Kubernetes Versions | Untested, but may work on Kubernetes Versions |
-|-----------------|-----------------------|---------------------|----------------------|-------------------------------|-----------------------------------------------|
-| master          | No, development only  |                     |                      |                               |                                               |
-| 1.10            | Yes                   | May 18, 2021        | ~Nov 2021 (Expected) | 1.18, 1.19, 1.20, 1.21        | 1.16, 1.17                                    |
-| 1.9             | Yes                   | February 9, 2021    | Aug 18, 2021         | 1.17, 1.18, 1.19, 1.20        | 1.15, 1.16                                    |
-| 1.8             | Yes                   | November 10, 2020   | May 12, 2021         | 1.16, 1.17, 1.18, 1.19        | 1.15                                          |
-| 1.7             | No                    | August 21, 2020     | Feb 25, 2021         | 1.16, 1.17, 1.18              | 1.15                                          |
-| 1.6             | No                    | May 21, 2020        | November 23, 2020    | 1.15, 1.16, 1.17, 1.18        |                                               |
-| 1.5 and earlier | No                    |                     |                      |                               |                                               |
+| Version         | Currently Supported   | Release Date        | End of Life          | Supported Kubernetes Versions | Untested, but may work |
+|-----------------|-----------------------|---------------------|----------------------|-------------------------------|------------------------|
+| master          | No, development only  |                     |                      |                               |                        |
+| 1.11            | Yes                   | August 2021 (Expect)| ~Feb 2022 (Expected) | 1.19, 1.20, 1.21, 1.22        | 1.16, 1.17, 1.18       |
+| 1.10            | Yes                   | May 18, 2021        | ~Nov 2021 (Expected) | 1.18, 1.19, 1.20, 1.21        | 1.16, 1.17             |
```

Comment:
I assume we should add 1.22 to the "untested but may work" version, and maybe also add Kubernetes 1.21 to Istio 1.9 ?

Reply:
Looking at the jobs, 1.9 is only tested up to 1.19, and 1.10 (and 1.11) have not tested on 1.22.0 GA but still on 1.22.beta. But agree we should add it to 1.10 as well.